script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -379,5 +379,5 @@ I reach the first floor.
 [sha:b1d0ba364a53c10222c6f4f749883ace7ee5c4e3]{
 -- Page 447, Offset 9776.
 -- 俺はナイフを握り直して、出来たばかりの爆心地に走り出した。
-I adjust my grip on my knife and run toward the newly formed crater.
+I adjust the grip on my knife and run toward the newly formed crater.
```

Comment:
Adjusting _the_ grip on the knife makes it sounds like he's diddling with the makeup of the handle. He's adjusting _his_ hands/action. If you don't like duplicate his I'd change the knife not his grip